prov/gni/src/gnix_fabric.c

```diff
@@ -363,14 +366,19 @@ static int gnix_getinfo(uint32_t version, const char *node, const char *service,
 				goto err;
 			}
 
-			if (hints->ep_attr->tx_ctx_cnt > 1) {
+			if (hints->ep_attr->tx_ctx_cnt > GNIX_SEP_MAX_CNT) {
```

Comment:
Sung or Howard can chime in here. If I recall correctly, the hints are provided to the provider and then the provider returns a fi_info structure reporting the actual limits of the provider. It would be the responsibility of the application to check the reported info from the provider than relying on the data from the original hints structure. 

If I'm wrong, then this sounds fine. Otherwise, if the application is truly responsible for honoring the values that we provide back  during fabric open, then it would be better for us to massage the value and let the fabric open succeed, rather than returning an error and possibly causing the application to abort because we confused them.